src/middleware/html-lang-middleware.ts

```diff
@@ -1,12 +1,14 @@
 import { NextFunction, Request, Response } from "express";
+import { PATH_NAMES } from "../app.constants";
 
 export function setHtmlLangMiddleware(
   req: Request,
   res: Response,
   next: NextFunction
 ): void {
   if (req.i18n) {
-    res.locals.htmlLang = req.i18n.language;
+    const re = new RegExp(`^${PATH_NAMES.CONTACT_US}`);
+    res.locals.htmlLang = re.test(req.url) ? "en" : req.i18n.language;
```

Comment:
Thanks @dbes-gds - I'm happy to take your judgment on this but I'd thought we still needed it because the `lang` attribute needs to reflect `req.i18n.language` for pages that aren't contact forms. My intention here was to ensure `lang="en"` for contact forms, even if the `req.i18n.language` is `"cy"`

Reply:
yes you're right :)